src/interfaces.ts

```diff
@@ -12,6 +12,8 @@ export interface Options {
   numa: boolean
   num_ctx: number
   num_batch: number
+  num_gpu: number
+  num_gqa: number
```

Comment:
```suggestion
```

We left this one out because Ollama should handle it, we will be deprecating it in the Ollama API soon. `num_gpu` should definitely be here though.